### ISSUE TITLE
Error when naming a single phenotype

### ISSUE THREAD

**Issue by agilly** [rgcgithub/regenie#68]
Not really a major bug, but I noticed that when the phenotype file contains a single phenotype, and that phenotype is named using `--phenoCol` or `--phenoColList`, I get the following error:

```
 * phenotypes       : [pheno.file] ERROR: Need at least one phenotype.
```

This also happens if the covar and pheno files are the same and I specify a single phenotype column. I got around this by looking at the code that said all non-FID/IID columns are used by default. So I created a pheno file with just FID, IID and the phenotype, without specifying `--phenoCol`, and it ran. The same command with `--phenoCol` set to `my_phenotype` crashes.

Version: REGENIE v1.0.6.7

**Comment by agilly** [comment]
Just had an epiphany and looked at the pheno file using `od -c`. It is Windows-coded, `\r\n`. I think Regenie tries to match the phenotype name including the carriage return `my_phenotype\r`. When I remove carriage returns, it runs fine. 

Which brings me to another question: will the phenotypes be affected, as in, does regenie understand that `0\r` is 0 and `1\r` is 1? I have running regenie jobs which I would need to restart if that's not the case.

Thanks,

Arthur

**Comment by joellembatchou** [comment]
Hi Arthur,

The conversion to double will not be affected by the "\r" present at the end of the number in the input file (so "0\r" will correctly be read as 0 and same for "1\r").

Cheers,
Joelle

**Comment by agilly** [comment]
Thanks Joelle, that's good to know !

Maybe it could be useful to add a quick line to remove carriage returns from a read file, e.g. `mystring.erase( std::remove(mystring.begin(), mystring.end(), '\r'), mystring.end() );` or perhaps make the error message clearer "could not match phenotype A with ones found in file (A, B, C, D,...). That may at least push the users in the right direction ;)

**Comment by joellembatchou** [comment]
Hi,

This has been fixed in commit 5379eb3b8e82cfa6d0afc4c64aa8e26ce5b716fa.

Cheers,
Joelle
